spec/decorators/user_decorator_spec.rb

```diff
@@ -20,35 +20,22 @@
   end
 
   describe "#cached_followed_tags" do
-    let(:tag1)  { create(:tag) }
-    let(:tag2)  { create(:tag) }
-    let(:tag3)  { create(:tag) }
-
-    it "returns empty if no tags followed" do
-      expect(saved_user.decorate.cached_followed_tags.size).to eq(0)
-    end
+    let(:saved_user) { create(:user) }
+    let(:tag1) { create(:tag) }
+    let(:tag2) { create(:tag) }
+    let(:tag3) { create(:tag) }
 
     it "returns array of tags if user follows them" do
       saved_user.follow(tag1)
       saved_user.follow(tag2)
       saved_user.follow(tag3)
-      expect(saved_user.decorate.cached_followed_tags.size).to eq(3)
-    end
-
-    it "returns tag object with name" do
-      saved_user.follow(tag1)
-      expect(saved_user.decorate.cached_followed_tags.first.name).to eq(tag1.name)
-    end
-
-    it "returns follow points for tag" do
-      saved_user.follow(tag1)
-      expect(saved_user.decorate.cached_followed_tags.first.points).to eq(1.0)
-    end
 
-    it "returns adjusted points for tag" do
-      follow = saved_user.follow(tag1)
-      follow.update(explicit_points: 0.1)
-      expect(saved_user.decorate.cached_followed_tags.first.points).to eq(0.1)
+      results = saved_user.decorate.cached_followed_tags
+      expect(results.size).to eq(3)
+      expect(results.first.points).to eq(1)
+      # Just a quick and dirty verification that we have a to_json method on an Array of Struct
+      # objects
+      expect(results.to_json).to be_present
```

Comment:
Based on the comment here, I don't think this expectation is doing what you think. We don't need to provide a `.to_json` method. The `json` library defines `Object#to_json`, so anything inheriting from `Object` will have it defined, even if it's garbage, making it virtually impossible for this expectation to fail:

```
➜  ~ pry -rjson
[1] pry(main)> [Object.new].to_json
=> "[\"#<Object:0x00007f9076aeb308>\"]"
```

This will pass a `be_present` matcher.

Reply:
It's a quick test that missed the mark.  I'll refactor.